client/foundries.go

```diff
@@ -845,18 +862,22 @@ func (a *Api) TargetsListRaw(factory string) (*[]byte, error) {
 	return a.Get(url)
 }
 
-func (a *Api) TargetsList(factory string) (*tuf.SignedTargets, error) {
-	body, err := a.TargetsListRaw(factory)
+func (a *Api) TargetsList(factory string, version ...string) (tuf.Files, error) {
```

Comment:
Interesting way to handle an optional parameter in Go :)

Reply:
I'd say, it's standard :)